bin/proxy.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-console */
+require('babel-register');
+
+const http = require('http');
```

Comment:
I think that preps everything for the next import. So any file that this file `require`s can use `import`, but this file still can't.